src/eradiate_plugins/phase/rayleigh_polarized.cpp

```diff
@@ -35,34 +36,40 @@ class RayleighPolarizedPhaseFunction final
     : public PhaseFunction<Float, Spectrum> {
 public:
     MI_IMPORT_BASE(PhaseFunction, m_flags)
-    MI_IMPORT_TYPES(PhaseFunctionContext)
+    MI_IMPORT_TYPES(PhaseFunctionContext, Volume)
 
     RayleighPolarizedPhaseFunction(const Properties &props) : Base(props) {
-        m_depolarization = props.get<ScalarFloat>("depolarization", 0.f);
+        m_depolarization = props.volume<Volume>("depolarization",0.f);
 
-        if (m_depolarization < 0.f || m_depolarization >= 1.f)
+        if( m_depolarization->max() >= 1.f )
```

Comment:
```suggestion
        if (m_depolarization->max() >= 1.f)
```

Also: what is the reason to drop the check on the minimum value?

Reply:
Will the formatting in a minute. Unfortunately, `Volume` only exposes its max value and not its min value so it is not possible to check for it anymore.